src/workers/parser/steps.js

```diff
@@ -1,30 +1,30 @@
 import { Source } from "../../../flow-typed/debugger-html";
 import { AstPosition } from "./types";
 import { getClosestPath } from "./utils/closest";
-import { isAwaitExpression } from "./utils/helpers";
+import { isAwaitExpression, isYieldExpression } from "./utils/helpers";
 import type { NodePath } from "babel-traverse";
 
 export function getNextStep(source: Source, pausedPosition: AstPosition) {
-  const awaitExpression = getAwaitExpression(source, pausedPosition);
+  const awaitExpression = getAsyncExpression(source, pausedPosition);
```

Comment:
this variable should also be renamed for consistency

Reply:
this should be totally renamed i now realize. it should probably be `currentExpression`